Popularity/Kipras/pyspark/rollOnSpark.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
```

Comment:
Kipras, we can do both ways and originally I said that we don't need to
merge, transform on Spark. I still think this is not necessary, since
if all data on HDFS in proper format we'll use them for training the model.
So what we need is set of tools which will put data into HDFS in a format
we need. In other words the series of steps for Spark would be:
- transform locally
- put data to HDFS
- run ML on Spark

We will not need merge step since we'll read all necessary files using
Spark APIs, moreover files can be read by multplie nodes on a cluster.

On Fri, Jul 29, 2016 at 8:27 AM, Kipras <notifications@github.com> wrote:

> In Popularity/Kipras/pyspark/rollOnSpark.sh
> <https://github.com/dmwm/DMWMAnalytics/pull/28#discussion_r72783351>:
>
> > @@ -0,0 +1,249 @@
> > +#!/bin/bash
>
> I believe I had this question before. I asked whether I have to rewrite
> all the merge, transform ect. functions on spark and you said that we only
> want to perform classification part on spark. So how is it ?
>
> Or maybe I don't understand something and you mean that we can run
> existing transform merge ect scripts on data that is on HDFS?
>
> Please clarify a bit this part.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dmwm/DMWMAnalytics/pull/28/files/bca227570064b0ea8e823e4ff9ae497274ced250#r72783351>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHo0tWoDglFYO_b4ZG8x8ABnznS9GBtks5qafGfgaJpZM4JXFlB>
> .
>